prody/proteins/interactions.py

```diff
@@ -0,0 +1,2030 @@
+           'calcHydrophohic', 'calcMetalInteractions',
+           'calcHydrogenBondsDCD', 'calcSaltBridgesDCD',
+           'calcRepulsiveIonicBondingDCD', 'calcPiStackingDCD', 
+           'calcPiCationDCD', 'calcHydrophohicDCD',
+           'calcProteinInteractions', 'calcStatisticsInteractions',
+           'compareInteractions', 
+           'calcLigandInteractions', 'listLigandInteractions', 
+           'showProteinInteractions_VMD', 'showLigandInteraction_VMD', 
+           'addHydrogens', 'calcHydrogenBondsDCD',
+           'Interactions', 'InteractionsDCD']
+
+
+def cleanNumbers(listContacts):
+    """Provide short list with indices and value of distance"""
+    
+    shortList = [ [int(str(i[0]).split()[-1].strip(')')), 
+                           int(str(i[1]).split()[-1].strip(')')), 
+                           str(i[0]).split()[1], 
+                           str(i[1]).split()[1], 
+                           float(i[2])] for i in listContacts ]    
+    
+    return shortList
+
+
+def calcPlane(atoms):
+    """Function provide paramaters of a plane for aromatic rings (based on 3 points).
+    Used in calcPiStacking()"""
+    
+    coordinates = atoms.getCoords()
+    p1, p2, p3 = coordinates[:3] # 3 points will be enough to obtain the plane
+    x1, y1, z1 = p1
+    x2, y2, z2 = p2
+    x3, y3, z3 = p3    
+    vec1 = p3 - p1 # These two vectors are in the plane
+    vec2 = p2 - p1
+    cp = np.cross(vec1, vec2) # the cross product is a vector normal to the plane
+    a, b, c = cp
+    d = np.dot(cp, p3) # This evaluates a * x3 + b * y3 + c * z3 which equals d
+    
+    return a,b,c,d
+
+
+def calcAngleBetweenPlanes(a1, b1, c1, a2, b2, c2):  
+    """Find angle between two planes"""
+    import math 
+          
+    d = ( a1 * a2 + b1 * b2 + c1 * c2 ) 
+    eq1 = math.sqrt( a1 * a1 + b1 * b1 + c1 * c1) 
+    eq2 = math.sqrt( a2 * a2 + b2 * b2 + c2 * c2) 
+    d = d / (eq1 * eq2) 
+    AngleBetweenPlanes = math.degrees(math.acos(d)) 
+    
+    return AngleBetweenPlanes
+    
+    
+def removeDuplicates(list_of_interactions):
+    ls=[]
+    newList = []
+    for no, i in enumerate(list_of_interactions):
+       i = sorted(list(array(i).astype(str)))
+       if i not in ls:
+           ls.append(i)
+           newList.append(list_of_interactions[no])
+    return newList
+
+
+def addHydrogens(pdb, method='openbabel', pH=7.0):    
+    """Function will add hydrogens to the protein and ligand structure using Openbabel [NO11]_
+    or PDBFixer with OpenMM.
+    
+    :arg pdb: PDB file name
+    :type pdb: str
+
+    :arg method: Name of program which will be use to fix protein structure.
+            Two alternative options are available: 'openbabel' and 'pdbfixer'.
+            For either option additional software need to be installed:
+            'openbabel': OpenBabel
+            'pdbfixer': PDBFixer and OpenMM
+    :type method: str, by default is 'openbabel'
```

Comment:
I don't think this will work. `:type:` field is supposed to be for type and nothing else. Please move default up to a new line before it